pyanaconda/modules/payloads/payload/dnf/dnf_manager.py

```diff
@@ -707,9 +705,14 @@ def _run_transaction(base, display):
         try:
             base.do_transaction(display)
             exit_reason = "DNF done"
+            for tsi in base.transaction:
+                if tsi.state == libdnf.transaction.TransactionItemState_ERROR:
+                    log.error("The transaction item %s ended in error state.", tsi)
+                    exit_reason = "The transaction has ended with some items in error state."
         except BaseException as e:  # pylint: disable=broad-except
-            log.error("The transaction has ended abruptly: %s", str(e))
-            exit_reason = str(e) + traceback.format_exc()
+            msg = "The transaction process has ended abruptly: " + str(e)
+            log.error(msg)
+            exit_reason = msg + traceback.format_exc()
```

Comment:
I think you want a `\n` between the two strings here?

Reply:
Added.